### PR TITLE
MC-16534: Add bccEmails to billing settings doc

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
#### Changes made
- I added a field to the billing settings called bccEmails.
- During the automatic payment job (upon failure of payment) or when sending an email to a customer for an expiring credit card, we add those emails as BCC.

#### Related PRs
- Core: https://github.com/cloudops/cloudmc-core/pull/1660
- UI: https://github.com/cloudops/cloudmc-ui/pull/2035

### CODE REVIEW

File: source/includes/administration/_billing_settings.md [hunk 1]
```diff
@@ -45,6 +46,7 @@ Attributes | &nbsp;
 `daysBeforeAutoDraft`<br/>*integer* | The number of days after the billing date to continue gathering missing customer usage before the invoice is drafted. Cannot be less than 2 days.
 `daysBeforeAutoApproval`<br/>*integer* | The number of days to review a draft invoice before it is automatically issued to the customer.
 `daysBeforeAutoPayment`<br/>*integer* | The number of days after the invoice has been issued before the customer's credit card is automatically charged.
+`bccEmails`<br/>*Array[string]* | List of email addresses to include as BCC when sending email to customers upon billing exceptions.
```
Comment (evanluc) [inline source/includes/administration/_billing_settings.md]:
```suggestion
`bccEmails`<br/>*Array[string]* | List of email addresses to include as BCC when sending emails to customers upon billing exceptions.
```

File: source/includes/administration/_billing_settings.md [hunk 2]
```diff
@@ -93,6 +96,7 @@ Attributes | &nbsp;
...
```
Comment (evanluc) [inline source/includes/administration/_billing_settings.md]:
```suggestion
`bccEmails`<br/>*Array[string]* | List of email addresses to include as BCC when sending emails to customers upon billing exceptions.
```

File: source/includes/administration/_billing_settings.md [hunk 3]
```diff
@@ -162,6 +168,7 @@ Optional | &nbsp;
 `customerInformation`<br/>*Array[string]* | The list of client custom fields to display in the invoice.
 `address`<br/>*Array[string]* | The address to display in the invoice. Each address field will be a row in the address block.
 `termsAndConditions`<br/>*string* | The terms and conditions to display in the invoice.
+`bccEmails`<br/>*Array[string]* | List of email addresses to include as BCC when sending email to customers upon billing exceptions.
```
Comment (evanluc) [inline source/includes/administration/_billing_settings.md]:
```suggestion
`bccEmails`<br/>*Array[string]* | List of email addresses to include as BCC when sending emails to customers upon billing exceptions.
```

File: source/includes/administration/_billing_settings.md [hunk 4]
```diff
@@ -232,6 +241,7 @@ Optional | &nbsp;
...
```
Comment (evanluc) [inline source/includes/administration/_billing_settings.md]:
```suggestion
`bccEmails`<br/>*Array[string]* | List of email addresses to include as BCC when sending emails to customers upon billing exceptions.
```

**Review by evanluc — APPROVED** [review]

**Review by patrickspensieri — APPROVED** [review]
Looks good aside from issue Evan pointed out

File: source/includes/administration/_billing_settings.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_billing_settings.md]: 6957fe5ccdfdeb31487bf3c1c5133f0a256014b0

File: source/includes/administration/_billing_settings.md (hunk 2 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_billing_settings.md]: 6957fe5ccdfdeb31487bf3c1c5133f0a256014b0

File: source/includes/administration/_billing_settings.md (hunk 3 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_billing_settings.md]: 6957fe5ccdfdeb31487bf3c1c5133f0a256014b0

File: source/includes/administration/_billing_settings.md (hunk 4 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_billing_settings.md]: 6957fe5ccdfdeb31487bf3c1c5133f0a256014b0

**Pull request merged**